content/en/blog/2022/SIG-Storage Spotlight.md

```diff
@@ -0,0 +1,86 @@
+
+## Ongoing challenges
+
+**FSM**: Specifically for the [1.25 release](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.25) we can see that there are a relevant number of storage-related [KEPs](https://bit.ly/k8s125-enhancements) in the pipeline, would you say that this release is particularly important for the SIG?
+
+**XY**: I wouldn’t say one release is more important than other releases. In any given release, we are working on a few very important things.
+
+**FSM**: Indeed, but are there any 1.25 specific specificities and highlights you would like to point out though?
+
+**XY**: Yes. For the 1.25 release, I want to highlight the following:
+
+* [CSI Migration](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration) is an on-going effort that SIG Storage has been working on for a few releases now. The goal is to move in-tree volume plugins to out-of-tree CSI drivers and eventually remove the in-tree volume plugins.  There are 7 KEPs that we are targeting in 1.25 are related to CSI migration. There is one core KEP for the general CSI Migration feature. That is targeting GA in 1.25. CSI Migration for GCE PD and AWS EBS are targeting GA. CSI Migration for vSphere is targeting to have the feature gate on by default while staying in 1.25 that are in Beta. Ceph RBD and PortWorx are targeting Beta, with feature gate off by default. Ceph FS is targeting Alpha.
+* The second one I want to highlight is [COSI, the Container Object Storage Interface](https://github.com/kubernetes-sigs/container-object-storage-interface-spec). This is a sub-project under SIG Storage. COSI proposes object storage Kubernetes APIs to support orchestration of object store operations for Kubernetes workloads. It also introduces gRPC interfaces for object storage providers to write drivers to provision buckets. The COSI team has been working on this project for more than two years now. The COSI feature is targeting Alpha in 1.25. The KEP just got merged. The COSI team is working on updating the implementation based on the updated KEP.
+* Another feature I want to mention is [CSI Ephemeral Volume](https://github.com/kubernetes/enhancements/issues/596) support. This feature allows CSI volumes to be specified directly in the pod specification for ephemeral use cases. They can be used to inject arbitrary states, such as configuration, secrets, identity, variables or similar information, directly inside pods using a mounted volume.  This was initially introduced in 1.15 as an alpha feature, and it is now targeting GA in 1.25.
```

Comment:
How about hyperlinking to https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volumes instead of https://github.com/kubernetes/enhancements/issues/596?

A link to the KEP _as well_ could be OK; eg, the text “targeting GA” could link to https://github.com/kubernetes/enhancements/issues/596#issuecomment-1161257318

Reply:
That's sensible, changed.